catalog/ui/catalog-ui-search/src/main/webapp/component/input/textarea/input-textarea.view.js

```diff
@@ -16,31 +16,60 @@
 var Marionette = require('marionette');
 var template = require('./input-textarea.hbs');
 var InputView = require('../input.view');
+var $ = require('jquery');
 
 module.exports = InputView.extend({
     template: template,
     events: {
-        'click .ta-expand': 'isOverflowing'
+        'click .ta-expand': 'expand',
+        'click .ta-contract': 'contract'
+    },
+    handleValue: function() {
+        this.$el.find('textarea').text(this.model.getValue());
+        this.hasOverflowed();
+    },
+    onRender: function () {
+        InputView.prototype.onRender.call(this);
+        this.hasOverflowed();
+        this.addResizeHandler();
+    },
+    onAttach: function() {
+        InputView.prototype.onAttach.call(this);
+        this.hasOverflowed();
     },
     getCurrentValue: function () {
         return this.$el.find('textarea').val();
     },
-    isOverflowing: function () {
+    hasOverflowed: function() {
```

Comment:
Is this related?  Or just another clean-up that is sneaking in?

Reply:
I had to go touch up the styling around notes and couldn't help myself.  I noticed we always show the more button even when there isn't anything more to show.  So I went and fixed it.  

There are more changes we should make to notes in the future to make it more consistent with the rest of the UI, but we can save those for a future PR.